internal/util/multi_error.go

```diff
@@ -5,20 +5,25 @@ import (
 	"strings"
 )
 
+// MultiError is a list of error.
```

Comment:
"MultiError is a collection of errors"

List is an implementation detail which is not exposed